core/protocols/aws-xml-protocol/src/main/java/software/amazon/awssdk/protocols/xml/internal/marshall/SimpleTypePathMarshaller.java

```diff
@@ -37,7 +37,7 @@
      * so that it preserves the path structure.
      */
     public static final XmlMarshaller<String> GREEDY_STRING =
-        new SimplePathMarshaller<>(ValueToStringConverter.FROM_STRING, PathMarshaller.GREEDY);
+        new SimplePathMarshaller<>(ValueToStringConverter.FROM_STRING, PathMarshaller.GREEDY_WITH_SLASHES);
```

Comment:
What services are affected by this change? How certain are we that they wouldn't be broken by this?

Reply:
This only effects XML services of which none of them currently use the greedy path marshaller.

Reply:
Other than S3 that is.